### PR TITLE
Setting up linter

### PR DESCRIPTION
**Pull request by emiliazm**
This is my first Microverse project.
In this project, I follow the git-flow and I set a linter.
I also ignore a test.md file.
Lastly, I am learning how to request a pull.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+test.md
+sth.md
+node_modules/
```
Comment (OLIPLICHE) [inline .gitignore:3]: Great Job here 👏 !!! You did well to add `test.md` and `node_modules` inside `.gitignore` file. but you pushed the `test.md file` and this is not part of the [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md)

- [ ] Please make sure that `test.md` is not present in your GitHub repository.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,8 @@
 # Hello-Microverse
 Setting up linters
+
+Built With
+Html and css.
+
+Getting Started.
+In this project we are learning to follow the git flow and set up linter.
```
Comment (OLIPLICHE) [inline README.md]: - [x] Please add a descriptive README file to your project - Kindly use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project ✔️

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @emiliazm👋, 

Good job so far!
There are some issues that you still need to work on to get your project approved, but you are almost there!
image


## Highlights (Excellent work)
Great 👏 👏👏!!!!!!!!!!
- Linter is set up correctly ✔️ 
- Linter result is green ✔️
- Github flow is correctly used ✔️
- You added .html and .css files in separate commit. ✔️


## Required Changes ♻️

- [ ] Please add a descriptive README file to your project
- [ ] Please your Pull Request should have a summary describing introduced changes in more detail
- [ ] Please capitalize your commit messages and use Imperative tense.

- [ ] Check the comments under the review. ⬇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (emiliazm) [reply README.md]: I add a descriptive README to my project. I used the temple from Microverse and I improve my file.

File: .gitignore (hunk 1 above)
Comment (emiliazm) [reply .gitignore:3]: Thanks for your comments. The test.md file appears on the .gitignore file. This is ok, right?

**Comment by emiliazm** [comment]
Hello @OLIPLICHE  thank you for your comments.
I made the following changes:
- I add a more descriptive README file.
- I made new commits capitalize and imperative.
- I checked the test.md file is not present on GitHub repository.

**Review by prolajumokeoni — APPROVED** [review]
Hi @emiliazm, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**